docs/source/guide/rem-5-theory.md

```diff
@@ -12,3 +12,29 @@ kernelspec:
 ---
 
 # What is the theory behind REM?
+
+Readout error mitigation (REM), in general, is probably one of the most general
+ and earliest error mitigation techniques, which itself can refer to a number
+ of specific approaches.
+
+Readout error mitigation can include simple postselection of bitstrings. For
+example, if one knows that the measured bitstrings should preserve some
+symmetry, bitstrings that do not preserve it, can be discarded. This
+possibility is indeed available in {mod}`mitiq.rem.post_select`.
```

Comment:
```suggestion
A simple version of readout error mitigation is postselection of bitstrings. For
example, if one knows that the measured bitstrings should preserve some
symmetry, bitstrings that do not preserve it can be discarded. Such capability 
is indeed available in {mod}`mitiq.rem.post_select`.
```